lib/routes/bloomberg/utils.ts

```diff
@@ -114,7 +114,10 @@ const parseArticle = (item) =>
                 }
 
                 // Blocked by PX3, or 404 by both api and direct link, return the default
-                const redirectUrls = res.redirectUrls.map(String);
+                let redirectUrls: string[] = [];
+                if (res.redirectUrls && res.redirectUrls.length > 0) {
```

Comment:
`res.redirectUrls` will always be `undefined` since #14922. `'@/utils/got'` is now a partial-got-like wrapper of [ofetch](https://github.com/unjs/ofetch).

Reply:
Ah, that explains why this broke recently. What's the recommended migration? Use `res.headers["Location"]` instead?